### PR TITLE
Clarify tutorial creates a new msg type

### PR DESCRIPTION
**Pull request by gavanderhoorn**
I realise there are more things to change to fully implement this change, but I wanted to first ask whether the suggestion makes sense.

"create a micro-ROS type" sounded like support for a new board was being added, or something else which was "a micro-ROS type".

From the tutorial it appears a custom message is created and a package to host it, which is then added to the micro-ROS build and eventually built.

Is this correct?

### CODE REVIEW

**Comment by pablogs9** [comment]
Any ideas @ralph-lange, @FranFin ?

**Comment by ralph-lange** [comment]
Yes, 'message type' is used commonly as generic term for message types, services types, and action types together. In this context, I typically speak of '_custom_ message types'. Therefore, I propose to rename the tutorial even to 'How to create custom message types for micro-ROS?' or just 'Custom message types with micro-ROS'.

**Comment by gavanderhoorn** [comment]
> Therefore, I propose to rename the tutorial even to 'How to create custom message types for micro-ROS?' or just 'Custom message types with micro-ROS'

that was going to be my suggestion as well.

But I first wanted to make sure I understood the tutorial correctly.

File: _docs/tutorials/core/create_new_type/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: How to create a new micro-ROS type
+title: How to create a new micro-ROS message type
```
Comment (pablogs9) [inline _docs/tutorials/core/create_new_type/index.md:2]:
```suggestion
title: How to create custom message types for micro-ROS
```

Comment (pablogs9) [reply _docs/tutorials/core/create_new_type/index.md:2]: @ralph-lange if you are ok, commit this suggestion and I'll merge

Comment (gavanderhoorn) [inline _docs/tutorials/core/create_new_type/index.md:2]:
```suggestion
title: Adding support for custom ROS message types to micro-ROS
```

Comment (gavanderhoorn) [reply _docs/tutorials/core/create_new_type/index.md:2]: I believe this would be more descriptive and accurate than the other two suggestions.

Comment (ralph-lange) [reply _docs/tutorials/core/create_new_type/index.md:2]: This title is a bit long in my opinion. In principle, the support is always there. It's really about creating a custom message type in an own project.

Comment (FranFin) [inline _docs/tutorials/core/create_new_type/index.md:2]:
```suggestion
title: How to add support for custom ROS message types to micro-ROS
```

**Review by FranFin — COMMENTED** [review]
@pablogs9 @ralph-lange @gavanderhoorn What do you think?

File: _docs/tutorials/core/create_new_type/index.md (hunk 1 above)
Comment (pablogs9) [reply _docs/tutorials/core/create_new_type/index.md:2]: ok

**Review by pablogs9 — APPROVED** [review]

File: _docs/tutorials/core/create_new_type/index.md (hunk 1 above)
Comment (gavanderhoorn) [reply _docs/tutorials/core/create_new_type/index.md:2]: This seems similar to my suggestion, which @ralph-lange wasn't happy with.

Perhaps "Including a custom ROS message in micro-ROS"?

That doesn't imply special support is needed (although I believe with the way ROS 2 and micro-ROS work, it is actually almost like adding support), and does seem to communicate that *those* are the steps you'd need to get your own custom messages (or someone else's) included in your local micro-ROS build.

Comment (ralph-lange) [reply _docs/tutorials/core/create_new_type/index.md:2]: I like you proposal, @gavanderhoorn! It is short, precise and already implies in the title that the procedure is similar to that for custom message types in normal ROS 2.

Comment (gavanderhoorn) [reply _docs/tutorials/core/create_new_type/index.md:2]: Ok, so here it is as a suggestion:

```suggestion
title: Including a custom ROS message in micro-ROS
```

Comment (gavanderhoorn) [reply _docs/tutorials/core/create_new_type/index.md:2]: Only thing is, it doesn't mention services nor actions, so I could see some users getting confused when reading this, but not sure we can help that other than making it clear in the tutorial text itself this is the procedure to follow for those as well.

Comment (pablogs9) [reply _docs/tutorials/core/create_new_type/index.md:2]: I'm ok with @gavanderhoorn suggestion. I assume that @ralph-lange also is. Please @FranFin commit and merge if you are ok.

**Comment by gavanderhoorn** [comment]
Friendly ping.

**Comment by gavanderhoorn** [comment]
Just checked the new site: were the changes in this PR reverted or did some branches work on the same files without those changes getting integrated? The current site seems to show the old title still.

**Comment by FranFin** [comment]
I think that the merged commit was not the one containing the final title proposal. What do you think about this https://github.com/micro-ROS/micro-ROS.github.io/pull/302? @gavanderhoorn @ralph-lange @pablogs9

**Comment by ralph-lange** [comment]
https://github.com/micro-ROS/micro-ROS.github.io/pull/302 looks good to me!

**Pull request merged**